languages/csharp/exercises/concept/classes/.docs/introduction.md

```diff
@@ -0,0 +1,66 @@
+The primary object-oriented construct in C# is the _class_, which is a grouping of related data (_fields_) and behavior (_methods_). The fields and methods of a class are known as its _members_.
```

Comment:
This is being _really_ picky here, but if it's grouping related data then that makes me wonder what are collections for? The following wording might make the difference:
```suggestion
The primary object-oriented construct in C# is the _class_, which is a grouping of different related pieces of data (_fields_) and behavior (_methods_). The fields and methods of a class are known as its _members_.
```

Still reads a bit odd to me, perhaps @iHiD will have a better handle on the grammatical angle.

Reply:
I'd say "a combination of data (fields) and behaviour (methods)". Removing "related" makes it clearer I think and I think related is implicit. I found this sentence tricky to read too when i first read it, but I got it enough not to comment, but you're right to say some people might not.

Reply:
I like the "combination" suggestion. I've updated.